Bottom: 3d27c62b5c96ca5e258bed7644c9fba410a84005
Top:    a02b585a40ca7479ac68787c95081f3aff295d66
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-07-16 15:09:05 -0500

Refresh of add-host-builder

---
diff --git a/ansible-playbook/host_vars/builder.novatech-llc.com/docker.yml b/ansible-playbook/host_vars/builder.novatech-llc.com/docker.yml
new file mode 100644
index 00000000..3c45191d
--- /dev/null
+++ b/ansible-playbook/host_vars/builder.novatech-llc.com/docker.yml
@@ -0,0 +1 @@
+worker_backend_network: backend
